### PR TITLE
WebAPI: Update Method pages to modern structure (part 34)

### PR DESCRIPTION
**Pull request by OnkarRuikar**
### Summary
Updates empty `### Return value` sections.
And puts `Return value` section after `Parameters` section, where it is not.

#### Metadata
- [x] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/web/api/document/execcommand/index.md [hunk 1]
```diff
@@ -56,7 +46,7 @@ disabled.
     providing that information. For example, `insertImage` requires the URL of
     the image to insert. Specify `null` if no argument is needed.
 
-### Commands
+#### Commands
```
Comment (wbamberg) [inline files/en-us/web/api/document/execcommand/index.md:49]: Not sure if this is in scope, but I think this should be a nested `<dl>` under `aCommandName`, like:

```
- `aCommandName`
  - : A string specifying the name of the command to execute. The following commands are specified:
    - `backColor` etc
```

**Review by wbamberg — CHANGES_REQUESTED** [review]
Looks great, just one suggestion.

File: files/en-us/web/api/document/execcommand/index.md (hunk 1 above)
Comment (OnkarRuikar) [reply files/en-us/web/api/document/execcommand/index.md]: done!

**Comment by OnkarRuikar** [comment]
With this PR we've completed adding/updating empty `Parameters` and `Return value` sections in all the eligible method pages.

Next PR will start updating parameter lists.

File: files/en-us/web/api/document/execcommand/index.md [hunk 2]
```diff
@@ -33,191 +33,126 @@ The [Clipboard API](/en-US/docs/Web/API/Clipboard_API) can be used instead of `e
 execCommand(aCommandName, aShowDefaultUI, aValueArgument)
 ```
 
+### Parameters
+
+- `aCommandName`
+  - : A string specifying the name of the command to execute.
```
Comment (OnkarRuikar) [inline files/en-us/web/api/document/execcommand/index.md:39]:
```suggestion
  - : A string specifying the name of the command to execute. The following commands are specified:
```

**Review by wbamberg — APPROVED** [review]
Epic.

**Pull request merged**